common/generate_bundle_base

```diff
@@ -51,7 +51,7 @@ fi
 if [ -n "${MASTER_OPTS[MODEL_CONSTRAINTS]}" ]; then
     juju set-model-constraints ${MASTER_OPTS[MODEL_CONSTRAINTS]}
 else
-    juju set-model-constraints root-disk-source=volume
+    juju set-model-constraints root-disk-source=volume root-disk=20G
```

Comment:
i think this makes sense for the general case. Just need to make sure that vms that do actually need the extra capacity define it.

Reply:
I agree, however it was mostly already working before we switched to root-disk-as-volume only a couple of weeks ago. So I think we're fine to ship this default and fix anything that comes up separately, unless you know of a specific case that needs fixing ahead of time.

The most obvious cases that come to mind are probably microk8s/kubernetes and maybe? mysql.

Reply:
I should actually quickly check, some instance types may have gotten more disk accidentally in the past due to flavor selection for ram or cores. I'll check on that.

Reply:
I tested this out, for a ceph and octavia deployment the only machine that got more cores/ram/storage by accident of prodstack6 flavors was nova-compute, so I pushed a new revision which also specifies cores=2 to ensure we still get the same.